controllers/drplacementcontrol_controller_test.go

```diff
@@ -261,6 +261,9 @@ func updateManagedClusterViewStatusAsNotFound(mcv *fndv2.ManagedClusterView) {
 
 	Eventually(func() bool {
 		err := k8sClient.Get(context.TODO(), mcvLookupKey, mcvLatest)
+		if errors.IsNotFound(err) {
```

Comment:
Fix is good. Question is why did we add the get post the update here? (asking for clarification for the future)

Reply:
Right, to make sure that the update is seen before we return from this function.  I have seen it where you make an update, the error returned is not nil but when you read, you still read the old object.  I am not sure if the envtest that is making the eventual consistency worse or not. So I aimed for protection.